src/Apps/Artist/Routes/Shows/index.tsx

```diff
@@ -61,15 +61,15 @@ export const ShowsRouteFragmentContainer = createFragmentContainer(ShowsRoute, {
         pastShowsStatus: { type: "String", defaultValue: "closed" }
         pastShowsSort: { type: "PartnerShowSorts", defaultValue: END_AT_DESC }
       ) {
-      artist_currentShows: artist(id: $artistID) {
+      artist_currentShows: artist(id: $artist_id) {
```

Comment:
Had to update this form because this variable name is passed verbatim directly to relay and to mp via found, etc. The filter needed it to match what our filter_artworks query params were expecting.